BMS_Reciever/BMS_Receiver.cpp

```diff
@@ -0,0 +1,80 @@
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/foreach.hpp>
+#include "BMS_Receiver.h"
+
+using boost::property_tree::ptree;
+
+void BMS_Receiver::get_sender_data(SENDER sender)
+{
+	switch (sender)
+	{
+	case CONSOLE:
+		get_data_from_console();
+		break;
+	case CONTROLLER:
+		//out of scope
```

Comment:
Dont keep out of scope 'dead code' in the commit. Someone else maintaining it will not know it's there.